### ISSUE TITLE
SPIKE: How to become aware of security incidents on the platform

### ISSUE THREAD

**Issue by davidkelliott** [ministryofjustice/modernisation-platform#1776]
## User Story

As a Modernisation Platform Engineer
I want to be alerted of security incidents reported by AWS tools
So that I can respond to them in a timely fashion

## Value

How do we make sure that we are aware of any security incidents that happen on the platform?
Can we leverage existing tools such as security hub?

We need to be alerted to critical/high security warnings from AWS security hub, whilst also being able to filter out the criticals which we have dismissed.

## Definition of done

- [x] review current alerting being sent into Slack Alerting channels
  - [ ] EG. SecurityHub into low-priority-alerts
- [x] sources of potential information identified (eg. SecurityHub)
- [x] possible methods for notifying the team on an alert identified
- [x] new issue(s) for implementation created

## Reference

[How to write good user stories](https://www.gov.uk/service-manual/agile-delivery/writing-user-stories)

**Comment by sukeshreddyg** [comment]
I reviewed the ticket, and we are not receiving any Security Hub alerts in the Slack alerting channel. The alerts we are getting in the low priority channel are all from CloudWatch metric filters.

**Comment by sukeshreddyg** [comment]
I discussed the findings and solution options with the team. The architecture to implement the solution has been finalized. Based on these discussions and findings, I have raised a new ticket to implement the proposed solution and complete the integration.

[Findings Document](https://docs.google.com/document/d/1j5h5RJcV-vPQ14viNKVGJJZ0t2zwCGGWEixR1cAU21g)
[New Implementation Ticket](https://github.com/ministryofjustice/modernisation-platform/issues/8076)
